docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,98 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
-
-## Example external application
-
-We'll be using [samltest.id](https://samltest.id/) to create a test consumer of
-Teleport's SAML identity provider.
-
-## Step 1/4. Configure a Teleport role with access to SAML service provider objects
-
-First, ensure you are logged into Teleport as a user that has permissions
-to read and modify `saml_idp_service_provider` objects. The default `editor` role
-has access to this already, but in case you are using a more customized configuration,
-create a role called `sp-manager.yaml` with the following contents:
-
+  Reference](./saml-reference.mdx) before proceeding.
+- User with preset `editor` role. We recommend creating a dedicated role to manage
```

Comment:
This point is a little bit confusing to me. We list the preset editor role as a perquisite (which means you must have it in order to continue), but then we immediately say "we recommend creating a dedicated role [and not using the preset]".

Can we be more explicit as to what we recommend?

Reply:
Move the role creation block to the end of the guide https://github.com/gravitational/teleport/pull/36467/commits/20806d4d4e6d30df09e5ec9f756d329f988623b7

This should address your concern here and below.